src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -5965,11 +5973,11 @@ namespace HVACUnitarySystem {
 						ShowContinueError( "...Fan inlet node name           = " + NodeID( FanInletNode ) );
 						ErrorsFound = true;
 					}
-					if ( FanOutletNode != UnitarySystem( UnitarySysNum ).UnitarySystemOutletNodeNum && UnitarySystem( UnitarySysNum ).FanExists ) {
+					if ( FanOutletNode != UnitarySystem( UnitarySysNum ).AirOutNode && UnitarySystem( UnitarySysNum ).FanExists ) {
 						ShowSevereError( CurrentModuleObject + " = " + UnitarySystem( UnitarySysNum ).Name );
 						ShowContinueError( "When a draw through fan is specified, the fan outlet node name must be the same as the unitary system outlet node name." );
 						ShowContinueError( "...Fan outlet node name           = " + NodeID( FanOutletNode ) );
-						ShowContinueError( "...UnitarySystem outlet node name = " + NodeID( UnitarySystem( UnitarySysNum ).UnitarySystemOutletNodeNum ) );
+						ShowContinueError( "...UnitarySystem outlet node name = " + NodeID( UnitarySystem( UnitarySysNum ).AirOutNode ) );
```

Comment:
Wow tons of these changes are just changing the variable being used in function calls and evaluations.

Reply:
Yes, variable names were unified across models. Who would have thought we should have been doing this all along.